apps/server/src/modules/room/domain/do/room.do.ts

```diff
@@ -0,0 +1,54 @@
+import { AuthorizableObject, DomainObject } from '@shared/domain/domain-object';
+import { EntityId } from '@shared/domain/types';
+
+export interface RoomProps extends AuthorizableObject {
+	id: EntityId;
+	name: string;
+	color: string;
+	startDate?: Date;
+	untilDate?: Date;
+}
+
+export class Room extends DomainObject<RoomProps> {
+	public getProps(): RoomProps {
+		const copyProps = { ...this.props };
+		// Note: Propagated hotfix. Will be resolved with mikro-orm update. Look at the comment in board-node.do.ts.
+		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+		// @ts-ignore
+		copyProps.domainObject = undefined;
+
```

Comment:
We could maybe write this as
```typescript
const { ...copyProps, domainObject } = this.props;
```
This would remove the "blacklisted" property from the copy instead of setting it to `undefined`.